modules/statistics/AvgRTPStatsReporter.js

```diff
@@ -564,14 +567,22 @@ export default class AvgRTPStatsReporter {
      */
     _calculateAvgStats(data) {
 
+        if (!data) {
+            logger.error('No stats');
+
+            return;
+        }
+
+        if (!data.transport || !data.transport.length) {
+            logger.error('No transport.');
```

Comment:
There are things other than transport being calculated here, in such case only transport related stuff should be omitted (values will be NaN and not reported).
This code is unaware when the transport will be unavailable - can not assume that lack of transport means lack of all other stats.